packages/wallet/frontend/src/components/cards/AccountCard.tsx

```diff
@@ -19,10 +19,20 @@ export const AccountCard = ({ account, idOnboarding }: AccountCardProps) => {
     const snapshotAccount = accountsSnapshot.find(
       (item) => item.assetCode === account.assetCode
     )
+    const baseAssetScale = 2
+    const maxAssetScale = 9
+
+    const snapshotBalance = snapshotAccount
+      ? Number(snapshotAccount.balance)
+      : 0
+    const accountBalance =
+      Number(account.balance) * Math.pow(10, maxAssetScale - account.assetScale)
+    const value = (snapshotBalance || accountBalance).toString()
+
```

Comment:
This could be done differently/cleaner but haven't put too much effort into it since the whole interface will be changed.